src/css/_docs.scss

```diff
@@ -418,12 +430,16 @@
     line-height: 17px;
     border: 1px solid $c-grey-mid;
     &:hover {
-      background-color: darken($c-background, 3%);
+      border: 1px solid $c-grey-light;
+      // background-color: darken($c-background, 3%);
     }
     &:before {
       color: $c-white;
       counter-increment: chapters;
       content: "Chapter " counter(chapters) ": ";
     }
   }
+  &-active {
+    border: 1px solid $c-white !important;
```

Comment:
`!important` is always a warning sign. If its the only thing that works, can we add a comment to explain why?